picard/webservice.py

```diff
@@ -404,10 +484,10 @@ def _run_next_task(self):
         if delay < sys.maxsize:
             self._timer_run_next_task.start(delay)
 
-    def add_task(self, func, host, port, priority, important=False):
-        hostkey = (host, port)
-        prio = int(priority)  # priority is a boolean
-        if important:
+    def add_task(self, func, wsrequest):
```

Comment:
I don't know of any plugin using `add_task` directly instead of going through the HTTP-method-specific functions, so this should be fine (maybe even rename this to `_add_task` to make it clear that it's not meant for public use).